src/main/java/net/sf/jabref/sql/DBConnectDialog.java

```diff
@@ -37,6 +37,11 @@
  */
 public class DBConnectDialog extends JDialog {
 
+    private static List<String> FORMATTED_NAMES = Arrays
```

Comment:
Hm, to me, this is a constant, and therefore should be static (and even made final as I see now).